### ISSUE TITLE
Enabling useBabelInstrumenter results in "Unknown option: direct.includePolyfill" error

### ISSUE THREAD

**Issue by lolmaus** [kategengler/ember-cli-code-coverage#18]
```
ReferenceError: [BABEL] app/adapters/application.js: Unknown option: direct.includePolyfill
    at Logger.error (/home/lolmaus/Code/launchpad2/node_modules/babel-core/lib/transformation/file/logger.js:58:11)
    at OptionManager.mergeOptions (/home/lolmaus/Code/launchpad2/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126:29)
    at OptionManager.init (/home/lolmaus/Code/launchpad2/node_modules/babel-core/lib/transformation/file/options/option-manager.js:216:10)
    at File.initOptions (/home/lolmaus/Code/launchpad2/node_modules/babel-core/lib/transformation/file/index.js:147:75)
    at new File (/home/lolmaus/Code/launchpad2/node_modules/babel-core/lib/transformation/file/index.js:137:22)
    at Pipeline.transform (/home/lolmaus/Code/launchpad2/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
    at Instrumenter.instrumentSync (/home/lolmaus/Code/launchpad2/node_modules/ember-cli-code-coverage/lib/babel-istanbul-instrumenter.js:36:45)
    at CoverageInstrumenter.processString (/home/lolmaus/Code/launchpad2/node_modules/ember-cli-code-coverage/lib/coverage-instrumenter.js:100:23)
    at CoverageInstrumenter.processFile (/home/lolmaus/Code/launchpad2/node_modules/broccoli-filter/index.js:165:31)
    at asyncProcessFile (/home/lolmaus/Code/launchpad2/node_modules/broccoli-filter/index.js:122:21)
    at lib$rsvp$$internal$$tryCatch (/home/lolmaus/Code/launchpad2/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (/home/lolmaus/Code/launchpad2/node_modules/rsvp/dist/rsvp.js:505:17)
    at /home/lolmaus/Code/launchpad2/node_modules/rsvp/dist/rsvp.js:1001:13
    at lib$rsvp$asap$$flush (/home/lolmaus/Code/launchpad2/node_modules/rsvp/dist/rsvp.js:1198:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)

The broccoli plugin was instantiated at: 
    at CoverageInstrumenter.Plugin (/home/lolmaus/Code/launchpad2/node_modules/broccoli-plugin/index.js:7:31)
    at CoverageInstrumenter.Filter [as constructor] (/home/lolmaus/Code/launchpad2/node_modules/broccoli-filter/index.js:34:10)
    at new CoverageInstrumenter (/home/lolmaus/Code/launchpad2/node_modules/ember-cli-code-coverage/lib/coverage-instrumenter.js:72:10)
    at CoreObject.module.exports.preprocessTree (/home/lolmaus/Code/launchpad2/node_modules/ember-cli-code-coverage/index.js:52:28)
    at /home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/broccoli/ember-app.js:584:41
    at Array.reduce (native)
    at EmberApp.addonPreprocessTree (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/broccoli/ember-app.js:583:30)
    at EmberApp.appAndDependencies (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/broccoli/ember-app.js:1062:18)
    at EmberApp.javascript (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/broccoli/ember-app.js:1199:34)
    at EmberApp.toArray (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/broccoli/ember-app.js:1604:10)
    at EmberApp.toTree (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/broccoli/ember-app.js:1626:30)
    at module.exports (/home/lolmaus/Code/launchpad2/ember-cli-build.js:35:14)
    at CoreObject.module.exports.Task.extend.setupBroccoliBuilder (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/models/builder.js:74:19)
    at CoreObject.module.exports.Task.extend.init (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/models/builder.js:54:10)
    at CoreObject.superWrapper [as init] (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/node_modules/core-object/lib/assign-properties.js:32:18)
    at CoreObject.Class (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/node_modules/core-object/core-object.js:32:33)
    at CoreObject.module.exports.Task.extend.run (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/tasks/build.js:15:19)
    at CoreObject.<anonymous> (/home/lolmaus/Code/launchpad2/node_modules/ember-cli/lib/commands/test.js:164:27)
    at lib$rsvp$$internal$$tryCatch (/home/lolmaus/Code/launchpad2/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (/home/lolmaus/Code/launchpad2/node_modules/rsvp/dist/rsvp.js:505:17)
    at /home/lolmaus/Code/launchpad2/node_modules/rsvp/dist/rsvp.js:1001:13
    at lib$rsvp$asap$$flush (/home/lolmaus/Code/launchpad2/node_modules/rsvp/dist/rsvp.js:1198:9)
```

**Comment by rwjblue** [comment]
Yes, we will need to strip that option out of the babel config before passing it on to babel.

See this being done in a few other places:

* https://github.com/babel/ember-cli-babel/blob/master/index.js#L82
* https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-app.js#L1183-L1188

**Comment by rwjblue** [comment]
Fix was released as v0.2.2
